src/Transition/CreateNewIDFUsingRulesV24_2_0.f90

```diff
@@ -485,6 +570,95 @@ SUBROUTINE CreateNewIDFUsingRules(EndOfFile,DiffOnly,InLfn,AskForInput,InputFile
+                CALL GetNewObjectDefInIDD(ObjectName, NwNumArgs, NwAorN, NwReqFld, NwObjMinFlds, NwFldNames, NwFldDefaults, NwFldUnits)
+                nodiff = .false.
+                OutArgs(1:13) = InArgs(1:13)
+                IF (SameString(InArgs(14), 'FAN:VARIABLEVOLUME')) THEN
+                  isVariableVolume = .TRUE.
+                  OutArgs(14) = 'Fan:SystemModel'
+                  OutArgs(15) = TRIM(InArgs(15))
+                  sysFanName = TRIM(InArgs(15))
+                ELSE
+                  OutArgs(14:15) = InArgs(14:15)
+                ENDIF
+                OutArgs(16:CurArgs) = InArgs(16:CurArgs)
+                CALL WriteOutIDFLines(DifLfn, 'ZoneHVAC:TerminalUnit:VariableRefrigerantFlow', CurArgs, OutArgs, NwFldNames, NwFldUnits)
+
+                IF (isVariableVolume) THEN
+                  ! create fan system model object
+                  ObjectName = 'Fan:SystemModel'
+                  DO Num3 = 1, NumFanVariableVolume
+                    IF (SameString(OldFanVO(Num3)%oldFanName, sysFanName)) THEN
+                      CALL GetNewObjectDefInIDD(ObjectName, NwNumArgs, NwAorN, NwReqFld, NwObjMinFlds, NwFldNames, NwFldDefaults, NwFldUnits)
+                      OutArgs(1) = TRIM(sysFanName)
+                      OutArgs(2) = OldFanVO(Num3)%availSchedule
+                      OutArgs(3) = OldFanVO(Num3)%inletAirNodeName
+                      OutArgs(4) = OldFanVO(Num3)%outletAirNodeName
+                      OutArgs(5) = OldFanVO(Num3)%maxAirFlow_str
+                      OutArgs(6) = 'Continuous'                             !- Speed Control Method
+                      IF (SameString(OldFanVO(Num3)%minFlowInputMethod, "FixedFlowRate")) THEN
+                        OutArgs(7) = OldFanVO(Num3)%maxAirFlow_str
+                      ELSE ! input method is "Fraction"
+                        READ(OldFanVO(Num3)%minAirFlowFrac_str, '(F15.5)') minAirFlowFrac
+                        IF (minAirFlowFrac == 0.0) THEN
+                          OutArgs(7) = '0.0'
+                        ELSE
+                          IF (.NOT. SameString(OldFanVO(Num3)%maxAirFlow_str, "AUTOSIZE")) THEN
+                            READ(OldFanVO(Num3)%maxAirFlow_str, '(F15.5)') maxAirFlow
+                            WRITE(OutArgs(7), '(F15.5)') (maxAirFlow * minAirFlowFrac)
+                          ELSE ! don't know how to multiply fraction with autosize
+                            OutArgs(7) = ''
+                          ENDIF
+                        ENDIF
+                      ENDIF
+                      OutArgs(8) = OldFanVO(Num3)%pressureRise_str          !- Design Pressure Rise {Pa}
+                      OutArgs(9) = OldFanVO(Num3)%motorEfficiency           !- Motor Efficiency
+                      OutArgs(10) = OldFanVO(Num3)%motorInAirStreamFrac     !- Motor In Air Stream Fraction
+                      IF (.NOT. SameString(OldFanVO(Num3)%maxAirFlow_str, "AUTOSIZE")) THEN
```

Comment:
No need to calculate a value for "Design Electric Power Consumption {W}" `OutArgs(11)`. It can be set to Autosize and then set the next field 12 "Design Power Sizing Method" to TotalEfficiencyAndPressure which is the method used by Fan:VariableVolume.
Set outargs 13 and 14 to blank.

Reply:
Note that the example files will need to be re-transitioned.

Reply:
Thanks. I just changed the `Design Electric Power Consumption` to use autosize and `Design Power Sizing Method` to `TotalEfficiencyAndPressure`. I'll also re-transition the example files